Dockerfile

```diff
@@ -191,3 +193,5 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vcs-url="https://github.com/nipreps/dmriprep" \
       org.label-schema.version=$VERSION \
       org.label-schema.schema-version="1.0"
+
+COPY license.txt /usr/local/license.txt
```

Comment:
what license is this?

Reply:
It's the Freesurfer license. Is there a better way to get that in there?

Reply:
Yup, there are a couple of options:
1. Use the ``--fs-license`` argument + mounting the file somewhere
2. Use the ``-e FS_LICENSE=/path/in/container/license.txt`` + mounting the file

Reply:
OK - I have removed this line.